rclip/utils/helpers.py

```diff
@@ -214,9 +217,14 @@ def download_image(url: str) -> Image.Image:
   if length := check_size.headers.get("Content-Length"):
     if int(length) > MAX_DOWNLOAD_SIZE_BYTES:
       raise ValueError(f"Avoiding download of large ({length} byte) file.")
-  img = Image.open(
-    cast(IO[bytes], requests.get(url, headers=headers, stream=True, timeout=DOWNLOAD_TIMEOUT_SECONDS).raw)
-  )
+  try:
+    img = Image.open(
+      cast(IO[bytes], requests.get(url, headers=headers, stream=True, timeout=DOWNLOAD_TIMEOUT_SECONDS).raw)
+    )
+  except DecompressionBombError:
+    raise DecompressionBombError(
```

Comment:
Not a blocking comment: maybe we can rethrow a custom error here instead of the PIL's `DecompressionBombError` to make it easy to differentiate between the two.